### PR TITLE
delete large log files before archiving to jenkins

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton @danmcp here's the change to delete the large files so they don't end up on jenkins, but i'm puzzled because the jenkins job looks like it doesn't archive the logs dir anyway today, only the artifacts dir.  did one of you change the job yesterday to temporarily prevent archiving the big files?

**Comment by bparees** [comment]
[test]

File: hack/test-end-to-end.sh [hunk 1]
```diff
@@ -93,6 +93,9 @@ function teardown()
     fi
     set -u
     set -e
+    # Clean up large log files so they don't end up on jenkins
+    find ${ARTIFACT_DIR} -size +20M -exec rm -f {} \;
```
Comment (smarterclayton) [inline hack/test-end-to-end.sh]: Does this output something to the log so we know that it got deleted?

Comment (bparees) [inline hack/test-end-to-end.sh]: it does now.

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
@smarterclayton re-review/merge?

**Comment by smarterclayton** [comment]
[merge] LGTM

**Comment by smarterclayton** [comment]
[merge]?

**Comment by smarterclayton** [comment]
Fixes #724

**Pull request merged**